base/inference.jl

```diff
@@ -2248,8 +2248,7 @@ function inlineable(f::ANY, e::Expr, atype::ANY, sv::StaticVarInfo, enclosing_as
     end
 
     body = Expr(:block)
-    body.args = filter(x->!((isa(x,Expr) && is(x.head,:line)) || isa(x,LineNumberNode)),
-                       ast.args[3].args::Array{Any,1})
+    body.args = ast.args[3].args::Array{Any,1}
     cost::Int = 1000
```

Comment:
possibly, it somewhat affects the expected average number of symbols per line (https://github.com/JuliaLang/julia/blob/d3af973d439e8f3628b63c7e305fdf5edf165ae2/base/inference.jl#L2624), but it's probably not too significant. the inliner doesn't want to inline long functions, and each `:line` Expr is pretty small. I think I'll wait to see if it is a concern.